package.json

```diff
@@ -49,7 +53,10 @@
     "net": "^1.0.2",
     "node-fetch": "^2.6.0",
     "web3": "^1.2.0",
-    "web3-provider-engine": "^15.0.3"
+    "web3-provider-engine": "^15.0.3",
+    "@openzeppelin/gsn-helpers": "^0.1.9",
```

Comment:
[Medium] I would suggest for @openzeppling packages to use fixed version instead as you never know when a minor version end up causing an incompatibility

Reply:
heh yea